airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalUtils.java

```diff
@@ -67,6 +68,15 @@ public static WorkflowOptions getWorkflowOptions(final TemporalJobType jobType)
         .build();
   }
 
+  public static ChildWorkflowOptions getChildWorkflowOptions(final TemporalJobType jobType) {
+    return ChildWorkflowOptions.newBuilder()
```

Comment:
We should add a deterministic workflow Id.

Reply:
n/a